geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/CreateDataSourceCommand.java

```diff
@@ -17,6 +17,7 @@
 import java.util.List;
 import java.util.Set;
 
+import org.springframework.shell.core.annotation.CliAvailabilityIndicator;
```

Comment:
Is this import required? I have the same question for the other changed files in this pull request.